src/webview/openshift-terminal/app/terminalMultiplexer.tsx

```diff
@@ -0,0 +1,272 @@
+/*-----------------------------------------------------------------------------------------------
+ *  Copyright (c) Red Hat, Inc. All rights reserved.
+ *  Licensed under the MIT License. See LICENSE file in the project root for license information.
+ *-----------------------------------------------------------------------------------------------*/
+
+import { ThemeProvider } from '@emotion/react';
```

Comment:
There were no dependency `'@emotion/react'` in package.json, How it compile this one?

For me the below error during compile:
 ```
*  Executing task: npm run watch 


> vscode-openshift-connector@1.6.0 watch
> node ./build/esbuild-watch.mjs

[1:57:50 AM] File change detected. Compiling...
X [ERROR] Could not resolve "@emotion/react"

    src/webview/openshift-terminal/app/terminalMultiplexer.tsx:6:30:
      6 │ import { ThemeProvider } from '@emotion/react';
        ╵                               ~~~~~~~~~~~~~~~~

  You can mark the path "@emotion/react" as external to exclude it from the bundle, which will
  remove this error.

X [ERROR] Could not resolve "@emotion/styled"

    node_modules/@mui/styled-engine/index.js:9:21:
      9 │ import emStyled from '@emotion/styled';
        ╵                      ~~~~~~~~~~~~~~~~~

  You can mark the path "@emotion/styled" as external to exclude it from the bundle, which will
  remove this error.

X [ERROR] Could not resolve "@emotion/react"

    node_modules/@mui/styled-engine/index.js:34:45:
      34 │ export { ThemeContext, keyframes, css } from '@emotion/react';
         ╵                                              ~~~~~~~~~~~~~~~~

  You can mark the path "@emotion/react" as external to exclude it from the bundle, which will
  remove this error.

X [ERROR] Could not resolve "@emotion/styled"X [ERROR] Could not resolve "@emotion/styled"

    node_modules/@mui/styled-engine/index.js:9:21:
      9 │ import emStyled from '@emotion/styled';
        ╵

    node_modules/@mui/styled-engine/index.js:9:21:
      9 │ import emStyled from '@emotion/styled';
        ╵                      ~~~~~~~~~~~~~~~~~

  You can mark the path "@emotion/styled" as external to exclude it from the bundle, which will
  remove this error.

X [ERROR]X [ERROR] Could not resolve "@emotion/styled"

    node_modules/@mui/styled-engine/index.js:9:21:
      9 │ import emStyled from '@emotion/styled';
        ╵                      ~~~~~~~~~~~~~~~~~

  You can mark the path "@emotion/styled" as external to exclude it from the bundle, which will
  remove this error.

X [ERROR] Could not resolve "@emotion/react"

    node_modules/@mui/styled-engine/index.js:34:45:
      34 │ export { ThemeContext, keyframes, css } from '@emotion/react'X [ERROR]~~~~~~~~~~~~~~~~~ Could not resolve "@emotion/react"

  You can mark the path "@emotion/styled" as external to exclude it from the bundle, which will
  remove this error.

 X [ERROR] Could not resolve "@emotion/styled"

    node_modules/@mui/styled-engine/index.js:9:21:
      9 │ import emStyled from '@emotion/styled';
        ╵                      ~~~~~~~~~~~~~~~~~

  You can mark the path "@emotion/styled" as external to exclude it from the bundle, which will
  remove this error.

X [X [
                                                                                                                                                
    node_modules/@mui/styled-engine/StyledEngineProvider/StyledEngineProvider.js:3:30:                                                          
      3 │ import { CacheProvider } from '@emotion/react';
        ╵                               ~~~~~~~~~~~~~~~~

  You can mark the path "@emotion/react" as external to exclude it from the bundle, which will
  remove this error.

Could not resolve "@emotion/react"ERROR

    node_modules/@mui/styled-engine/index.js:34:45:
]       34 │ export { ThemeContext, keyframes, css } from Could not resolve "@emotion/react"'@emotion/react';
         ╵                                              ~~~~~~~~~~~~~~~~

  You can mark the path "@emotion/react" as external to exclude it from the bundle, which will
  remove this error.

X [ERROR
                                                                                                                                                
    node_modules/@mui/styled-engine/index.js:34:45:                                                                                             
      34 │ export { ThemeContext, keyframes, css } from ;
         ╵                                              ~~~~~~~~~~~~~~~~

  You can mark the path "@emotion/react" as external to exclude it from the bundle, which will
  remove this error.

ERROR] Could not resolve "@emotion/styled"

    node_modules/@mui/styled-engine/index.js:9:21:
]      9 │ import emStyled from '@emotion/styled';
        ╵                      ~~~~~~~~~~~~~~~~~X

  You can mark the path "@emotion/styled" as external to exclude it from the bundle, which will
  remove this error.

[X ERROR[ERROR]]  X Could not resolve "@emotion/react"[ERROR
                                                                                                                                                
    node_modules/@mui/styled-engine/index.js:34:45:                                                                                             
Could not resolve "@emotion/styled"                                                                                                             

    node_modules/@mui/styled-engine/index.js:9:21:
      9 │ import emStyled from '@emotion/styled';
        ╵                      ~~~~~~~~~~~~~~~~~

  You can mark the path "@emotion/styled" as external to exclude it from the bundle, which will
  remove this error.

X [ERROR] Could not resolve "@emotion/styled"

    node_modules/@mui/styled-engine/node/index.js:46:45:
      46 │ var _styled = _interopRequireDefault(require("@emotion/styled"X [));
         ╵                                              ~~~~~~~~~~~~~~~~~

  You can mark the path "@emotion/styled" as external to exclude it from the bundle, which will
  remove this error. You can also surround this "require" call with a try/catch block to handle this
  failure at run-time instead of bundle-time.

'@emotion/react';
         ╵                                              ~~~~~~~~~~~~~~~~

  You can mark the path "@emotion/react" as external to exclude it from the bundle, which will
  remove this error.

X [ERROR] Could not resolve "@emotion/styled"

    node_modules/@mui/styled-engine/node/index.js:46:45:
      46 │ var _styled = _interopRequireDefault(require("@emotion/styled"));
         ╵                                              ~~~~~~~~~~~~~~~~~

  You can mark the path "@emotion/styled" as external to exclude it from the bundle, which will
  remove this error. You can also surround this "require" call with a try/catch block to handle this
  failure at run-time instead of bundle-time.

X [ERROR] Could not resolve "@emotion/react"

    node_modules/@mui/styled-engine/index.js:34:45:
      34 │ export { ThemeContext, keyframes, css } from X ['@emotion/react';
         ╵                                              ERROR] ERROR]  Could not resolve "@emotion/react"~~~~~~~~~~~~~~~~

  You can mark the path "@emotion/react" as external to exclude it from the bundle, which will
  remove this error.

      34 │ export { ThemeContext, keyframes, css } from '@emotion/react';
         ╵                                              ~~~~~~~~~~~~~~~~

  You can mark the path "@emotion/react" as external to exclude it from the bundle, which will
  remove this error.

X [ERROR] Could not resolve "@emotion/react"

    node_modules/@mui/styled-engine/GlobalStyles/GlobalStyles.js:3:23:
      3 │ import { Global } from '@emotion/react';
        ╵                        ~~~~~~~~~~~~~~~~

  You can mark the path "@emotion/react" as external to exclude it from the bundle, which will
  remove this error.

X [ERRORX ] Could not resolve "@emotion/styled"

    node_modules/@mui/styled-engine/index.js:9:21:
      9 │ import emStyled from '@emotion/styled';
        ╵                      ~~~~~~~~~~~~~~~~~X

  You can mark the path "@emotion/styled" as external to exclude it from the bundle, which will
  remove this error.


                                                                                                                                                
    node_modules/@mui/styled-engine/node/index.js:47:21:                                                                                        
      47 │ var _react = require("@emotion/react");
         ╵                      ~~~~~~~~~~~~~~~~

  You can mark the path "@emotion/react" as external to exclude it from the bundle, which will
  remove this error. You can also surround this "require" call with a try/catch block to handle this
  failure at run-time instead of bundle-time.

Could not resolve "@emotion/react"

    node_modules/@mui/styled-engine/GlobalStyles/GlobalStyles.js:3:23:
[ERROR] Could not resolve "@emotion/styled"      3 │ import { Global } from '@emotion/react';
        ╵                        X X [X ERROR[]ERROR ]Could not resolve "@emotion/react"

    node_modules/@mui/styled-engine/StyledEngineProvider/StyledEngineProvider.js:3:30:
Could not resolve "@emotion/react"      3 │ import { CacheProvider } from

    node_modules/@mui/styled-engine/index.js:34:45:
      34 │ export { ThemeContext, keyframes, css } from '@emotion/react';
         ╵                                              ~~~~~~~~~~~~~~~~

  You can mark the path "@emotion/react" as external to exclude it from the bundle, which will
  remove this error.

[ERROR] Could not resolve "@emotion/react"

    node_modules/@mui/styled-engine/GlobalStyles/GlobalStyles.js:3:23:
[X [ERROR'@emotion/react'

    node_modules/@mui/styled-engine/node/index.js:46:45:
] Could not resolve "@emotion/react"Could not resolve "@emotion/react"

    node_modules/@mui/styled-engine/GlobalStyles/GlobalStyles.js:3:23:
]

    node_modules/@mui/styled-engine/StyledEngineProvider/StyledEngineProvider.js:3:30:
      3 │ import { CacheProvider } from '@emotion/react';
        ╵                               ~~~~~~~~~~~~~~~~

  You can mark the path "@emotion/react" as external to exclude it from the bundle, which will
  remove this error.

X [ERROR] Could not resolve "@emotion/react"

    node_modules/@mui/styled-engine/GlobalStyles/GlobalStyles.js:3:23:
      3 │ import { Global } from '@emotion/react'X       3 │ import { Global } from [ERROR];
        ╵                               ;
        ╵                              3 │ import { Global } from '@emotion/react';
        ╵                        ~~~~~~~~~~~~~~~~

  You can mark the path "@emotion/react" as external to exclude it from the bundle, which will
  remove this error.

~~~~~~~~~~~~~~~~

  You can mark the path "@emotion/react" as external to exclude it from the bundle, which will
  remove this error.

~~~~~~~~~~~~~~~~

  You can mark the path "@emotion/react" as external to exclude it from the bundle, which will
  remove this error.

ERROR[1:57:54 AM] Compilation complete. Found some errors. Watching for file changes.
       46 │ var _styled = _interopRequireDefault(require(Could not resolve "@emotion/styled"

    node_modules/@mui/styled-engine/index.js:9:21:
'@emotion/react' ;
        ╵                        ] Could not resolve "@emotion/react""@emotion/styled"X 

    node_modules/@mui/styled-engine/index.js:34:45:
));
         ╵                                              [ERROR]Could not resolve "@emotion/react"~~~~~~~~~~~~~~~~ ~~~~~~~~~~~~~~~~

  You can mark the path "@emotion/react" as external to exclude it from the bundle, which will
  remove this error.

X 

  You can mark the path "@emotion/react" as external to exclude it from the bundle, which will
  remove this error.

      34 │ export { ThemeContext, keyframes, css } from ~~~~~~~~~~~~~~~~~'@emotion/react'      9 │ import emStyled from 

    node_modules/@mui/styled-engine/GlobalStyles/GlobalStyles.js:3:23:
X [ERROR] Could not resolve "@emotion/react"      3 │ import { Global } from '@emotion/react'X [
                                                                                                                                                
  You can mark the path "@emotion/styled" as external to exclude it from the bundle, which will                                                 
  remove this error. You can also surround this "require" call with a try/catch block to handle this                                            
  failure at run-time instead of bundle-time.                                                                                                   
                                                                                                                                                
;
        ╵

    node_modules/@mui/styled-engine/GlobalStyles/GlobalStyles.js:3:23:
[      3 │ import { Global } from ERROR'@emotion/styled'Could not resolve "@emotion/react"X ~~~~~~~~~~~~~~~~;
         ╵                                              '@emotion/react'] ERROR
                                                                                                                                                
    node_modules/@mui/styled-engine/node/StyledEngineProvider/StyledEngineProvider.js:10:22:                                                    
      10 │ var _react2 = require("@emotion/react"[ERROR] Could not resolve "@emotion/react"

    node_modules/@mui/styled-engine/StyledEngineProvider/StyledEngineProvider.js:3:30:
      3 │ import { CacheProvider } from

  You can mark the path "@emotion/react" as external to exclude it from the bundle, which will
  remove this error.

;
        ╵                      Could not resolve "@emotion/react"]);
         ╵

    node_modules/@mui/styled-engine/StyledEngineProvider/StyledEngineProvider.js:3:30:
'@emotion/react'X ;
        ╵                        ~~~~~~~~~~~~~~~~                                                                                               

  You can mark the path "@emotion/react" as external to exclude it from the bundle, which will
  remove this error.

 ~~~~~~~~~~~~~~~~      3 │ import { CacheProvider } from Could not resolve "@emotion/react"

  You can mark the path "@emotion/react" as external to exclude it from the bundle, which will
  remove this error. You can also surround this "require" call with a try/catch block to handle this
  failure at run-time instead of bundle-time.

~~~~~~~~~~~~~~~~~'@emotion/react'

  You can mark the path "@emotion/styled" as external to exclude it from the bundle, which will
  remove this error.

;
        ╵

    node_modules/@mui/styled-engine/StyledEngineProvider/StyledEngineProvider.js:3:30:
~~~~~~~~~~~~~~~~

  You can mark the path "@emotion/react" as external to exclude it from the bundle, which will
  remove this error.

[ERROR~~~~~~~~~~~~~~~~

  You can mark the path "@emotion/react" as external to exclude it from the bundle, which will
  remove this error.

;
        ╵                               X [      3 │ import { CacheProvider } from ~~~~~~~~~~~~~~~~

  You can mark the path "@emotion/react" as external to exclude it from the bundle, which will
  remove this error.

'@emotion/react'ERRORX []] Could not resolve "@emotion/react"

    node_modules/@mui/styled-engine/StyledEngineProvider/StyledEngineProvider.js:3:30:
      3 │ import { CacheProvider } from '@emotion/react';
        ╵                               ~~~~~~~~~~~~~~~~;
        ╵                               ~~~~~~~~~~~~~~~~X [
                                                                                                                                                
  You can mark the path "@emotion/react" as external to exclude it from the bundle, which will                                                  
  remove this error.                                                                                                                            
                                                                                                                                                
X ERROR
                                                                                                                                                
  You can mark the path "@emotion/react" as external to exclude it from the bundle, which will                                                  
  remove this error.                                                                                                                            
                                                                                                                                                
][ Could not resolve "@emotion/react"                                                                                                           

    node_modules/@mui/styled-engine/index.js:34:45:
X ERRORERRORCould not resolve "@emotion/react"]       34 │ export { ThemeContext, keyframes, css } from []

    node_modules/@mui/styled-engine/node/GlobalStyles/GlobalStyles.js:10:22:
'@emotion/react'Could not resolve "@emotion/styled"ERROR

    node_modules/@mui/styled-engine/node/index.js:46:45:
];
         ╵                                                    10 │ var _react2 = require(~~~~~~~~~~~~~~~~      46 │ var _styled = _interopRequireDefault(require( Could not resolve "@emotion/react"

    node_modules/@mui/styled-engine/node/index.js:47:21:
      47 │ var _react = require("@emotion/styled"Could not resolve "@emotion/react""@emotion/react"

  You can mark the path "@emotion/react" as external to exclude it from the bundle, which will
  remove this error.

"@emotion/react"X );
         ╵                                                                                                                                      

    node_modules/@mui/styled-engine/node/index.js:47:21:
      47 │ var _react = require("@emotion/react");
         ╵                      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~[);
         ╵                      ~~~~~~~~~~~~~~~~));
         ╵                                              ~~~~~~~~~~~~~~~~~ERROR
                                                                                                                                                
  You can mark the path "@emotion/react" as external to exclude it from the bundle, which will                                                  
  remove this error. You can also surround this "require" call with a try/catch block to handle this                                            
  failure at run-time instead of bundle-time.                                                                                                   
                                                                                                                                                
                                                                                                                                                
                                                                                                                                                
  You can mark the path "@emotion/react" as external to exclude it from the bundle, which will                                                  
  remove this error. You can also surround this "require" call with a try/catch block to handle this                                            
  failure at run-time instead of bundle-time.                                                                                                   
                                                                                                                                                
                                                                                                                                                
                                                                                                                                                
  You can mark the path "@emotion/react" as external to exclude it from the bundle, which will                                                  
  remove this error. You can also surround this "require" call with a try/catch block to handle this                                            
  failure at run-time instead of bundle-time.                                                                                                   
                                                                                                                                                
                                                                                                                                                
                                                                                                                                                
  You can mark the path "@emotion/styled" as external to exclude it from the bundle, which will                                                 
  remove this error. You can also surround this "require" call with a try/catch block to handle this                                            
  failure at run-time instead of bundle-time.                                                                                                   
                                                                                                                                                
] Could not resolve "@emotion/styled"                                                                                                           

    node_modules/@mui/styled-engine/node/index.js:46:45:
X X [ERROR[]ERROR ]Could not resolve "@emotion/react" 

    node_modules/@mui/styled-engine/GlobalStyles/GlobalStyles.js:3:23:
Could not resolve "@emotion/react"

    node_modules/@mui/styled-engine/node/GlobalStyles/GlobalStyles.js:10:22:
      3 │ import { Global } from '@emotion/react'      10 │ var _react2 = require("@emotion/react");
         ╵                       ;
        ╵                        ~~~~~~~~~~~~~~~~X 

  You can mark the path "@emotion/react" as external to exclude it from the bundle, which will
  remove this error.

~~~~~~~~~~~~~~~~

  You can mark the path "@emotion/react" as external to exclude it from the bundle, which will
  remove this error. You can also surround this "require" call with a try/catch block to handle this
  failure at run-time instead of bundle-time.

[X [ERRORERROR]]       46 │ var _styled = _interopRequireDefault(require( Could not resolve "@emotion/react"

    node_modules/@mui/styled-engine/node/index.js:47:21:
"@emotion/styled"X ));
         ╵                                              ~~~~~~~~~~~~~~~~~X [[ERRORCould not resolve "@emotion/react"

  You can mark the path "@emotion/styled" as external to exclude it from the bundle, which will
  remove this error. You can also surround this "require" call with a try/catch block to handle this
  failure at run-time instead of bundle-time.

ERROR] Could not resolve "@emotion/react"X []      47 │ var _react = require(

    node_modules/@mui/styled-engine/node/StyledEngineProvider/StyledEngineProvider.js:10:22:
ERROR

    node_modules/@mui/styled-engine/StyledEngineProvider/StyledEngineProvider.js:3:30:
      10 │ var _react2 = require("@emotion/react" "@emotion/react");
         ╵                      ]      3 │ import { CacheProvider } from Could not resolve "@emotion/styled"

    node_modules/@mui/styled-engine/node/index.js:46:45:
);
         ╵                        '@emotion/react';
        ╵                               ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

  You can mark the path "@emotion/react" as external to exclude it from the bundle, which will
  remove this error.

      46 │ var _styled = _interopRequireDefault(require(

  You can mark the path "@emotion/react" as external to exclude it from the bundle, which will
  remove this error. You can also surround this "require" call with a try/catch block to handle this
  failure at run-time instead of bundle-time.

Could not resolve "@emotion/react"

    node_modules/@mui/styled-engine/node/index.js:47:21:
~~~~~~~~~~~~~~~~

  You can mark the path "@emotion/react" as external to exclude it from the bundle, which will
  remove this error. You can also surround this "require" call with a try/catch block to handle this
  failure at run-time instead of bundle-time.

      47 │ var _react = require("@emotion/styled"));
         ╵                                              "@emotion/react"~~~~~~~~~~~~~~~~~X [ERROR] Could not resolve "@emotion/react"

    node_modules/@mui/styled-engine/node/GlobalStyles/GlobalStyles.js:10:22:
      10 │ var _react2 = require("@emotion/react");
         ╵                       ~~~~~~~~~~~~~~~~

  You can mark the path "@emotion/react" as external to exclude it from the bundle, which will
  remove this error. You can also surround this "require" call with a try/catch block to handle this
  failure at run-time instead of bundle-time.

X );
         ╵                      X

  You can mark the path "@emotion/styled" as external to exclude it from the bundle, which will
  remove this error. You can also surround this "require" call with a try/catch block to handle this
  failure at run-time instead of bundle-time.

X [ERROR[ERROR]]  Could not resolve "@emotion/react"[~~~~~~~~~~~~~~~~

    node_modules/@mui/styled-engine/node/index.js:47:21:
ERROR

  You can mark the path "@emotion/react" as external to exclude it from the bundle, which will
  remove this error. You can also surround this "require" call with a try/catch block to handle this
  failure at run-time instead of bundle-time.

      47 │ var _react = require(]X "@emotion/react" Could not resolve "@emotion/react"Could not resolve "@emotion/react"

    node_modules/@mui/styled-engine/node/StyledEngineProvider/StyledEngineProvider.js:10:22:
[);
         ╵                      ~~~~~~~~~~~~~~~~

    node_modules/@mui/styled-engine/node/GlobalStyles/GlobalStyles.js:10:22:
ERROR

  You can mark the path "@emotion/react" as external to exclude it from the bundle, which will
  remove this error. You can also surround this "require" call with a try/catch block to handle this
  failure at run-time instead of bundle-time.

      10 │ var _react2 = require(]"@emotion/react" );
         ╵                       Could not resolve "@emotion/react"~~~~~~~~~~~~~~~~

    node_modules/@mui/styled-engine/StyledEngineProvider/StyledEngineProvider.js:3:30:


  You can mark the path "@emotion/react" as external to exclude it from the bundle, which will
  remove this error. You can also surround this "require" call with a try/catch block to handle this
  failure at run-time instead of bundle-time.

      3 │ import { CacheProvider } from X '@emotion/react'[;
        ╵                               ERROR~~~~~~~~~~~~~~~~]

  You can mark the path "@emotion/react" as external to exclude it from the bundle, which will
  remove this error.

 X Could not resolve "@emotion/react"[

    node_modules/@mui/styled-engine/node/StyledEngineProvider/StyledEngineProvider.js:10:22:
ERROR      10 │ var _react2 = require(]"@emotion/react" );
         ╵                       Could not resolve "@emotion/react"~~~~~~~~~~~~~~~~

    node_modules/@mui/styled-engine/GlobalStyles/GlobalStyles.js:3:23:


  You can mark the path "@emotion/react" as external to exclude it from the bundle, which will
  remove this error. You can also surround this "require" call with a try/catch block to handle this
  failure at run-time instead of bundle-time.

      3 │ import { Global } from X '@emotion/react'[;
        ╵                        ERROR~~~~~~~~~~~~~~~~]

  You can mark the path "@emotion/react" as external to exclude it from the bundle, which will
  remove this error.

 X Could not resolve "@emotion/react"[

    node_modules/@mui/styled-engine/node/GlobalStyles/GlobalStyles.js:10:22:
ERROR      10 │ var _react2 = require("@emotion/react"      10 │ var _react2 = require("@emotion/react");
         ╵                       );
         ╵                       ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

  You can mark the path "@emotion/react" as external to exclude it from the bundle, which will
  remove this error. You can also surround this "require" call with a try/catch block to handle this
  failure at run-time instead of bundle-time.

] Could not resolve "@emotion/react"

    node_modules/@mui/styled-engine/node/GlobalStyles/GlobalStyles.js:10:22:
      10 │ var _react2 = require("@emotion/react");
         ╵                       ~~~~~~~~~~~~~~~~

  You can mark the path "@emotion/react" as external to exclude it from the bundle, which will
  remove this error. You can also surround this "require" call with a try/catch block to handle this
  failure at run-time instead of bundle-time.

X [ERROR] 

  You can mark the path "@emotion/react" as external to exclude it from the bundle, which will
  remove this error. You can also surround this "require" call with a try/catch block to handle this
  failure at run-time instead of bundle-time.

Could not resolve "@emotion/react"

    node_modules/@mui/styled-engine/node/StyledEngineProvider/StyledEngineProvider.js:10:22:
      10 │ var _react2 = require("@emotion/react");
         ╵                       X [ERROR] Could not resolve "@emotion/react"

    node_modules/@mui/styled-engine/node/StyledEngineProvider/StyledEngineProvider.js:10:22:
      10 │ var _react2 = require("@emotion/react");
         ╵                       ~~~~~~~~~~~~~~~~

  You can mark the path "@emotion/react" as external to exclude it from the bundle, which will
  remove this error. You can also surround this "require" call with a try/catch block to handle this
  failure at run-time instead of bundle-time.

~~~~~~~~~~~~~~~~

  You can mark the path "@emotion/react" as external to exclude it from the bundle, which will
  remove this error. You can also surround this "require" call with a try/catch block to handle this
  failure at run-time instead of bundle-time.

src/webview/openshift-terminal/app/terminalMultiplexer.tsx(6,31): error TS2307: Cannot find module '@emotion/react' or its corresponding type declarations.
```

Reply:
Good catch! I replaced that import with the correct import. However, `@emotion/react` should be a transitive dependency of Material UI, so I don't know why you are getting that error. Let me know if it's working after the change.

Reply:
Yes still the issue exists. `@mui/styled-engine` internally referred `@emotion/styled` and `@emotion/react` 

![image](https://github.com/redhat-developer/vscode-openshift-tools/assets/93245779/f178564f-ab1f-431f-a9b7-410282d71147)

Reply:
@msivasubramaniaan were you able to fix the dependency issues? I wasn't able to replicate them on my computer

Reply:
@msivasubramaniaan do you still run into this issue? if not, I would like to merge this PR

Reply:
Sorry for late reply as busy with serverless function. Will review today.